src/compiler/types/lines_5000_5250.rs

```diff
@@ -34,6 +34,16 @@ impl __String {
     pub fn chars(&self) -> std::str::Chars {
         self.0.chars()
     }
+
+    // TODO: should implement via Deref instead?
+    pub fn as_str(&self) -> &str {
+        &self.0
+    }
+
+    // TODO: should implement via some trait?
+    pub fn eq_str(&self, str: &str) -> bool {
```

Comment:
I don't know if there's a standard trait for doing "equality comparison" against a different type?